index.js

```diff
@@ -197,7 +198,7 @@ module.exports = {
     'operator-linebreak': 0,
     'padded-blocks': 0,
     'quote-props': 0,
-    'quotes': 0,
+    'quotes': [2, 'single'],
```

Comment:
👍 

https://eslint.org/docs/rules/quotes#enforce-the-consistent-use-of-either-backticks-double-or-single-quotes-quotes